tests/integration/docker/swarm.bats

```diff
@@ -74,6 +97,39 @@ setup() {
 	done 
 }
 
+@test "check that replicas has two interfaces" {
+	REPLICAS_UP=(`$DOCKER_EXE ps -a | tail -n +2 | awk '{ print $1 }'`)
+	for i in ${REPLICAS_UP[@]}; do
+		# here we are checking that each replica has two interfaces 
+		# and they should be always eth0 and eth1
+		$DOCKER_EXE exec $i bash -c "ip route show | grep -E eth0 && ip route show | grep -E eth1"
+	done
+}
+
+@test "check service ip among the replicas" {
+	service_name=`$DOCKER_EXE service ls | tail -n +2 |  awk '{ print $1 }'`
+	ip_service=`$DOCKER_EXE service inspect $service_name | grep Addr | cut -d':' -f2 | cut -d'"' -f2 |  cut -d'/' -f1`
+	REPLICAS_UP=(`$DOCKER_EXE ps -a | tail -n +2 | awk '{ print $1 }'`)
```

Comment:
I think using -a is incorrect as you get non running containers
This may be more accurate
$DOCKER_EXE service ps $service_name  --filter desired-state=running